kornia/augmentation/_2d/intensity/clahe.py

```diff
@@ -0,0 +1,63 @@
+from typing import Any, Dict, Optional, Tuple
+
+from kornia.augmentation import random_generator as rg
+from kornia.augmentation._2d.intensity.base import IntensityAugmentationBase2D
+from kornia.core import Tensor
+from kornia.enhance import equalize_clahe
+
+
+class RandomClahe(IntensityAugmentationBase2D):
+    r"""Apply CLAHE equalization on the input tensor randomly.
+
+    .. image:: _static/img/equalize_clahe.png
+
+    Args:
+        clip_limit: threshold value for contrast limiting. If 0 clipping is disabled.
+        grid_size: number of tiles to be cropped in each direction (GH, GW).
+        slow_and_differentiable: flag to select implementation
+        same_on_batch: apply the same transformation across the batch.
+        p: probability of applying the transformation.
+        keepdim: whether to keep the output shape the same as input (True) or broadcast it
+                 to the batch form (False).
+    .. note::
+        This function internally uses :func:`kornia.enhance.equalize_clahe`.
+
+    Examples:
+        >>> img = torch.rand(1, 10, 20)
+        >>> inv = RandomClahe()
+        >>> res = inv(img)
+        >>> res.shape
+        torch.Size([1, 10, 20])
```

Comment:
```suggestion
        torch.Size([1, 1, 10, 20])
```
and let's open an issue to investigate what's wrong here

Reply:
i think it's the normal behavior of augmentations, where it will return batched